### ISSUE TITLE
[BUG] `create-vue` playwright config fails in VS Code extension due to relying on npm script `PATH`

### ISSUE THREAD

**Issue by segevfiner** [microsoft/playwright#21059]
**Context:**
- Playwright Version: 1.30.0
- Operating System: macOS 13.2.1
- Node.js version: 16.19.0
- Visual Studio Code version: 1.75.1
- Playwright for VSCode extension version: 1.0.7
- Browser: All
- Extra: Project created from `create-vue`

**Code Snippet**

The default playwright config in `create-vue`, sets the following:
```ts
  /* Run your local dev server before starting the tests */
  webServer: {
    /**
     * Use the dev server by default for faster feedback loop.
     * Use the preview server on CI for more realistic testing.
    Playwright will re-use the local server if there is already a dev-server running.
     */
    command: process.env.CI ? 'vite preview --port 5000' : 'vite dev',
    port: 5000,
    reuseExistingServer: !process.env.CI,
  },
  ```

**Describe the bug**

This config fails in the VS Code plugin in a pnpm monorepo, but works from the CLI, due to not setting the `PATH` correctly for playwright to find `vite` from the sub-project `node_modules`.

**Comment by yury-s** [comment]
Try adding `pnpm exec` to the command in your config:

```ts
command: process.env.CI ? 'pnpm exec vite preview --port 5000' : 'pnpm exec vite dev',
```

the extension runs playwright tests by executing `node` binary, it doesn't know anything about `pnpm` and the paths it may setup. When you run from command line via pnpm, it will add `.bin` to the `PATH` which is why `vite` may be called directly.

**Comment by segevfiner** [comment]
It likely will. But since `{npm exec,yarn,pnpm} playwright test` works, it kinda makes sense for it to work the same when the extension launchs it so there are less surprises like that. The stock behavior is in `libnpmexec` I think, and there is the `npm-run-path` module might be simpler to use for this.

**Comment by melpeln** [comment]
you could also try using a different playwright config for CI and adding "--config playwright-ci.config.json"

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your contribution to our project. This issue has been closed due to its limited upvotes and recent activity, and insufficient feedback for us to effectively act upon. Our priority is to focus on bugs that reflect higher user engagement and have actionable feedback, to ensure our bug database stays manageable.

Should you feel this closure was in error, please create a new issue and reference this one. We're open to revisiting it given increased support or additional clarity. Your understanding and cooperation are greatly appreciated.
